### ISSUE TITLE
classifier AvgPool2d stride

### ISSUE THREAD

**Issue by Haoyanlong** [pytorch/vision#351]
The stride in AvgPool2d at model.classifier should be 13

self.classifier = nn.Sequential(
            nn.Dropout(p=0.5),
            final_conv,
            nn.ReLU(inplace=True),
            nn.AvgPool2d(13, stride=13)

**Comment by alykhantejani** [comment]
Hi @Haoyanlong, what model are you referencing?

**Comment by yannadani** [comment]
@Haoyanlong Are you referring to the squeezenet?

**Comment by Haoyanlong** [comment]
@alykhantejani @yannadani ,I reference the squeezenet,but I find the code has some difference with package in pytorch. so I debug the code,the output size of nn.Average's last layer is 1000*6*6, so to make the output's size 1000*1*1, nn.Average(kernel_size=6).Could give me some guild?Thank you!!!

**Comment by alykhantejani** [comment]
@Haoyanlong what is the input size to your network. I believe the expected input size is `224 x 224` @fmassa @soumith is this correct?

**Comment by fmassa** [comment]
With https://github.com/pytorch/vision/pull/643, squeezenet now uses `nn.AdaptiveAvgPool2d`, so this can be closed, as it now supports arbitrary input sizes.
